pyccel/codegen/printing/cwrappercode.py

```diff
@@ -528,14 +673,11 @@ def _print_Interface(self, expr):
 
         # Find a name for the wrapper function
         wrapper_name = self._get_wrapper_name(used_names, expr)
+        used_names.add(wrapper_name)
 
         # Collect local variables
```

Comment:
Do we still collect local variables somewhere?

Reply:
Sorry, my question was too brief. I meant that while the variables `python_func_args`, `python_func_kwargs` and `python_func_selfarg` have been removed in this PR, the comment line `# Collect local variables` has remained. Was this intentional?

Reply:
The lines have not been removed but have just been shifted to the function `get_wrapper_arguments` so this section still does the same thing. That said, I'm not sure this comment ever made sense. I have removed it

Reply:
See c21c497

Reply:
Gotcha. Thanks